pysat/tests/test_custom.py

```diff
@@ -251,8 +251,8 @@ def custom1(inst):
 
         self.testInst.custom.attach(custom1, 'add')
         self.testInst.load(2009, 1)
-        assert self.testInst.meta['doubleMLT'].units == 'hours1'
-        assert self.testInst.meta['doubleMLT'].long_name == 'doubleMLT'
+        assert self.testInst.meta[('doubleMLT', 'units')] == 'hours1'
+        assert self.testInst.meta[('doubleMLT', 'long_name')] == 'doubleMLT'
```

Comment:
```suggestion
        assert self.testInst.meta['doubleMLT', 'units'] == 'hours1'
        assert self.testInst.meta['doubleMLT', 'long_name'] == 'doubleMLT'
```

Reply:
Was it not working for you, or is this a style thing?